pkg/trace/sampler/remote_rates.go

```diff
@@ -134,18 +133,15 @@ func (r *RemoteRates) updateTPS(tpsTargets map[Signature]apmsampling.TargetTPS)
 
 // Start runs and adjust rates per signature following remote TPS targets
 func (r *RemoteRates) Start() {
-	go func() {
-		for update := range r.client.SamplingUpdates() {
-			r.onUpdate(update)
-		}
-		close(r.stopped)
-	}()
+	// Make sure the remote client is running, if the client is already started
+	// this is a nop so this is fine.
```

Comment:
Suggest removing "if the client is already started this is a nop so this is fine". The fact that this can be called multiple times with no side effects can be described in the godoc for `r.client.Start()` if it isn't already.